site/_data/popovers.yaml

```diff
@@ -92,6 +92,9 @@ pulsar:
 pulsar-functions:
   q: What are Pulsar Functions?
   def: Pulsar Functions are lightweight functions that can consume messages from Pulsar topics, apply custom processing logic, and, if desired, publish results to topics.
+reader:
+  q: What is a Pulsar reader?
+  def: A Pulsar reader is a Pulsar consumer that enables you to manually manage the topic cursor.
```

Comment:
I think that using the term "consumer" might be misleading here since the alternative to `Reader` is to use a `Consumer`. 

I think other point that could be mentioned here is that Reader does not retain data like consumer with a subscription